app/uk/gov/hmrc/trusts/controllers/GetTrustController.scala

```diff
@@ -65,7 +65,7 @@ class GetTrustController @Inject()(identify: IdentifierAction,
 
         case _: ResourceNotFoundResponse.type =>
           auditService.auditErrorResponse(TrustAuditing.GET_TRUST, Json.obj("utr" -> utr), request.identifier, "Not Found received from DES.")
-          NotFound
+          InternalServerError
```

Comment:
This should be returned to the old implementation. It should return a 404 NotFound.

Reply:
changed back to 404